tutorials/containers/deploying-storageos-on-kubernetes/deploying-storageos-on-kubernetes.md

```diff
@@ -0,0 +1,186 @@
+---
+id: deploying-storageos-on-kubernetes
+summary: Learn how to deploy StorageOS on Charmed Kubernetes
+categories: containers, kubernetes, csi, storageos
+tags: juju, kubernetes, beginner, container, storageos, cloud
+difficulty: 3
+status: draft
+published: 2019-09-16
+author: Peter De Sousa <peter.de.sousa@canonical.com>
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+
+---
+
+# Deploying StorageOS on Charmed Kubernetes
+
+## Overview
+Duration: 1:00
+
+### In this tutorial you'll learn how to...
+
+- Deploy the StorageOS operator
+- Create a secret for StorageOS
+
+
+### You will only need
+
+- Multipass [for Windows](6https://ubuntu.com/blog/kubernetes-on-windows-how-to-set-up) or [Mac](https://tutorials.ubuntu.com/tutorial/install-microk8s-on-mac-os#0) if using [MicroK8s](https://tutorials.ubuntu.com/tutorial/install-a-local-kubernetes-with-microk8s#0)
+- or a Charmed Kubernetes deployment
+
+
+## Installing StorageOS
+
+Make sure that your kubernetes-master is configured to allow privileged.
+
+`$ juju config kubernetes-worker allow-privileged=true`
```

Comment:
for code in docs:
 
- always use code fencing
- don't include prompt($ or #) 

e.g, this would become:

<pre>
```bash
juju config kubernetes-worker allow-privileged=true
```
</pre>

Reply:
Cheers!

Reply:
Cheers!